### ISSUE TITLE
Redirect lookit.mit.edu to childrenhelpingscience.com

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1352]
# TL; DR

The site is currently available via two domains: lookit.mit.edu and childrenhelpingscience.com. We should continue supporting the MIT domain but redirect all traffic to the CHS domain.

# Narrative

Following the Lookit/CHS merger, we are supporting both domains for the site but trying to re-brand as CHS. It's important that the MIT domain is still functional, but for branding consistency we would like the CHS domain to be used by researchers, in email templates, etc. 

Also, the use of two different domains is negatively affecting the participant experience in some cases, because logins are specific to a single domain. For example, if a participant completes a study on one domain (e.g. MIT) and is redirected back to the site after the study using the other domain (e.g. CHS), then the participant will hit the login page instead. 

Our best solution for this problem is to redirect all MIT-domain traffic to the CHS domain. This way, the MIT links will still be functional, but it will not be possible for a participant to be logged in to the MIT domain (since the login page will redirect to CHS) and there will be no uncertainty as to which domain a user might be on. 

# Implementation

The actual redirection should be largely done by Burwood. We will need to test things and address any unforeseen consequences.

- [x] Implement domain redirect on **staging**: https://github.com/lookit/lookit-orchestrator/pull/49. This will:
  - Redirect all requests for lookit-staging.mit.edu to babieshelpingscience.com  
  - Change the domain for the study links in emails from lookit-staging.mit.edu to babieshelpingscience.com (#1336)
  - Change the domain for a study's default exit URL from lookit-staging.mit.edu to babieshelpingscience.com (#1274)

Note that merging the PR will just set this change up in the orchestrator, but not actually cause it to go into production. After merging we will need to rebuild/deploy staging, test, and then rebuild/deploy production (or revert the orchestrator PR if there are problems).

- [ ] Implement domain redirect on **production**: (PR to come). This will:
  - Redirect all requests for lookit.mit.edu to childrenhelpingscience.com  
  - Change the domain for the study links in emails from lookit.mit.edu to childrenhelpingscience.com (#1336)
  - Change the domain for a study's default exit URL from lookit.mit.edu to childrenhelpingscience.com (#1274)

# To do

- [ ] Test user experience on staging (@ianchandlercampbell, @msheskin): https://github.com/lookit/lookit-api/issues/1355
- [ ] Test API - do we need to modify the API requests made by other services? (Pipe, EFP, AWS Lambda, others?)

# Other related issues that make sense to address at the same time

- [x] Change the default Exit URL from MIT domain to CHS (or variable from settings): #1274 - this relies on an updated value from orchestrator, so it will go into effect once this PR is merged into production and production is rebuilt/deployed.
- [x] Change domain in study links in emails: #1336 - this relies on a value from orchestrator, so it will happen at the same time as the redirection, without any changes to the site. It should already be in effect on staging, and will go into effect on production after production has been rebuilt/deployed.
- [ ] Update lingering use of MIT-domain URLs in the codebase (links in Terms of Service, in tests, etc.)

**Comment by becky-gilbert** [comment]
Reopening this issue (it was closed automatically because it's linked to the orchestrator PR that has been merged).
